pkg/envtest/komega/default.go

```diff
@@ -0,0 +1,85 @@
+package komega
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// defaultK is the Komega used by the package global functions.
+var defaultK = &komega{}
+
+// SetDefaultClient sets the client used by the package global functions.
+func SetDefaultClient(c client.Client) {
+	defaultK = &komega{client: c}
+}
+
+func checkDefaultClient() {
+	if defaultK.client == nil {
+		panic("Default Komega's client is not set. Use WithDefaultClient to set it.")
```

Comment:
```suggestion
		panic("Default Komega's client is not set. Use SetDefaultClient to set it.")
```
nit. Should this be `SetDefaultClient`?